.github/workflows/lint.yml

```diff
@@ -1,16 +1,17 @@
-name: Node CI
+name: Lint
 
 on: [push,pull_request]
 
 jobs:
   lint-js:
-    name: Lint Javascript/Typescript via eslint
+    name: JS & TS
```

Comment:
This is probably a question of personal choice - but I like having the actual program doing the enforcement visible. That way if a new contributor runs into issues they know what program to use to format immediately.

Reply:
One can open up the build to see what tool is being used to lint the code. If they would like to know what this tool is, they would most likely be looking at the entire workflow anyways, not just the title. What was previously done normally went offscreen and required someone to actually open up the workflow in a separate tab to read it.

README should probably be updated for this as a better replacement instead, as that's where contributors would first go to get their questions answered.

Reply:
I mean it renders fine for me (and this is having the tab in a half-window) 
![image](https://user-images.githubusercontent.com/5902494/124304941-66887a00-db32-11eb-8076-e597d9d41ff1.png)

There's also the other topic that changing the names of these things requires changing the protection hooks which normally wouldnt be annoying but this has happened several times now

Reply:
![image](https://share.alru.ga/aPg2)

Reply:
That's my issue, if one wants to know at a glance their workflows status, it shouldn't go offscreen. But if they want to know more about the build sequence, then they should normally hit the individual 'Details' button to dive further in-depth about it.